CTestConfig.cmake

```diff
@@ -0,0 +1,4 @@
+set(CTEST_PROJECT_NAME Kokkos Kernels)
+set(CTEST_NIGHTLY_START_TIME 01:00:00 UTC)
+set(CTEST_SUBMIT_URL https://my.cdash.org/submit.php?project=Kokkos+Kernels)
```

Comment:
Adding a note here, jobs run from SNL resources/hardware currently cannot post results to my.cdash.org - results can only be posted on approved internal server resources

Reply:
That is correct, the jobs planned will be coming from ALCF and OLCF only